src/context/capabilities.rs

```diff
@@ -266,16 +266,18 @@ pub unsafe fn get_capabilities(gl: &gl::Gl, version: &Version, extensions: &Exte
             0
         },
 
-        max_indexed_shader_storage_buffer: if version >= &Version(Api::Gl, 4, 3) {      // TODO: ARB_shader_storage_buffer_object   // TODO: GLES
-            let mut val = mem::uninitialized();
-            gl.GetIntegerv(gl::MAX_SHADER_STORAGE_BUFFER_BINDINGS, &mut val);
-            val
-        } else {
-            0
+        max_indexed_shader_storage_buffer: {
+            if version >= &Version(Api::Gl, 4, 3) || extensions.gl_arb_shader_storage_buffer_object {      // TODO: GLES
+                let mut val = mem::uninitialized();
+                gl.GetIntegerv(gl::MAX_SHADER_STORAGE_BUFFER_BINDINGS, &mut val);
+                val
+            } else {
+                0
+            }
         },
 
         max_indexed_transform_feedback_buffer: {
-            if version >= &Version(Api::Gl, 3, 0) || extensions.gl_arb_transform_feedback3 {      // TODO: make sure that GL 3.0 supports it   // TODO: GLES
+            if version >= &Version(Api::Gl, 4, 0) || extensions.gl_arb_transform_feedback3 {      // TODO: GLES
```

Comment:
Are you sure it appears in 3.0? I couldn't find it anywhere in https://www.opengl.org/registry/doc/glspec30.20080923.pdf, but it is in the 3.1 specification.